seqr/views/apis/variant_search_api.py

```diff
@@ -255,7 +255,7 @@ def export_variants_handler(request, search_hash):
     variants, _ = get_es_variants(results_model, page=1, load_all=True, user=request.user)
     variants = _flatten_variants(variants)
 
-    json, variants_to_saved_variants = _get_saved_variants(variants, families)
+    json_saved_variants, variants_to_saved_variants = _get_saved_variants(variants, families)
```

Comment:
this is fine to leave in, but in the future you really shouldn't put out-of-scope style changes into a PR as it just makes review harder